### PR TITLE
:bookmark: release 1.7.0

### PR DESCRIPTION
**Pull request by annashamray**
fixes https://github.com/maykinmedia/objects-api/issues/436 (ON part)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,27 +2,53 @@
 Changes
 =======
 
-1.7.0 (2024-??-??)
+1.7.0 (2024-08-26)
```
Comment (Coperh) [inline CHANGELOG.rst:5]: Date in the past

**Review by Coperh — APPROVED** [review]
Sentry environments have been added which could be include din the change log.

Otherwise looks like everything is included.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,13 +2,46 @@
...
 ------------------
 
 **New features**
 
-* Made user emails unique to prevent two users logging in with the same email, causing an error
+* [#169] Made user emails unique to prevent two users logging in with the same email, causing an error
+* [#151] Added 2FA to the Admin
+* [#157] Optimized deleting abonnement with a lot of notifications in the Admin
```
Comment (stevenbal) [inline CHANGELOG.rst:12]: Guess we could also add this:

* [maykinmedia/open-api-framework#21] display version in admin

**Review by stevenbal — APPROVED** [review]
Looks good! One minor feature that might also be listed, but otherwise it seems good

File: CHANGELOG.rst (hunk 1 above)
Comment (annashamray) [reply CHANGELOG.rst:5]: I'll fixed it

**Comment by annashamray** [comment]
@Coperh Added a line about sentry environment into the changelog

**Pull request merged**